care/users/signals.py

```
@@ -19,6 +23,11 @@ def password_reset_token_created(
     Handles password reset tokens
     When a token is created, an e-mail needs to be sent to the user
     """
+    # Add debugging logs
+    # logger.debug(f"Password reset token created for user: {reset_password_token.user.username}")
+    # logger.debug(f"Token key: {reset_password_token.key}")
+    # logger.debug(f"Reset URL: {settings.CURRENT_DOMAIN}/password_reset/{reset_password_token.key}")
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)